docs/Rules/UseCorrectCasing.md

```diff
@@ -10,35 +10,58 @@ title: UseCorrectCasing
 
 ## Description
 
-This is a style/formatting rule. PowerShell is case insensitive wherever possible,
-so the casing of cmdlet names, parameters, keywords and operators does not matter.
-This rule nonetheless ensures consistent casing for clarity and readability.
-Using lowercase keywords helps distinguish them from commands.
-Using lowercase operators helps distinguish them from parameters.
+This is a style/formatting rule. PowerShell is case insensitive wherever possible, so the casing of
+cmdlet names, parameters, keywords and operators does not matter. This rule nonetheless ensures
+consistent casing for clarity and readability. Using lowercase keywords helps distinguish them from
+commands. Using lowercase operators helps distinguish them from parameters.
 
 ## How
 
-Use exact casing for type names.
+- Use exact casing for type names.
+- Use exact casing of the cmdlet and its parameters.
+- Use lowercase for language keywords and operators.
 
-Use exact casing of the cmdlet and its parameters, e.g.
-`Invoke-Command { 'foo' } -RunAsAdministrator`.
+## Configuration
 
-Use lowercase for language keywords and operators.
+```powershell
+Rules = @{
+    PS UseCorrectCasing = @{
+        Enable        = $true
+        CheckCommands = $true
+        CheckKeyword  = $true
+        CheckOperator = $true
+    }
+}
+```
+
+### Parameters
+
+#### Enable: bool (Default value is `$false`)
+
+Enable or disable the rule during ScriptAnalyzer invocation.
+
+#### CheckCommands: bool (Default value is `$true`)
 
-## Example
+If true, require the case of all operators to be lowercase.
 
-### Wrong
+#### CheckKeyword: bool (Default value is `$true`)
+
+If true, require the case of all keywords to be lowercase.
+
+#### CheckOperator: bool (Default value is `$true`)
+
+If true, require the case of all commands to match their actual casing.
+
+## Examples
+
+### Wrong way
 
 ```powershell
-ForEach ($file IN get-childitem -recurse) {
-    $file.Extension -Eq '.txt'
-}
+invoke-command { 'foo' } -runasadministrator
 ```
 
-### Correct
+### Correct way
 
 ```powershell
-foreach ($file in Get-ChildItem -Recurse) {
-    $file.Extension -eq '.txt'
-}
+Invoke-Command { 'foo' } -RunAsAdministrator
```

Comment:
Can this have keyword examples like the previous did (e.g. `foreach` not `ForEach`)?

Reply:
Fixed